package.json

```diff
@@ -18,20 +18,20 @@
     "json-schema-generator": "./bin/cli.js"
   },
   "dependencies": {
```

Comment:
why are we upgrading a ton of other dependencies in this pr alone? should we just focus on replacing `request`?

Reply:
I guess I thought if I was running regression tests I might as well try to get the other dependencies up-to-date. I can remove these changes and patch it in another PR.